File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/PutIntoCashInventoryPacket.scala
package net.server.channel.packet.cash.operation

class PutIntoCashInventoryPacket(private var _action: Int, private var _cashId: Int, private var _invType: Byte) extends BaseCashOperationPacket(_action = _action) {
  def cashId: Int = _cashId

  def invType: Byte = _invType
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/PutIntoCashInventoryPacket.java
package net.server.channel.packet.cash.operation;

public class PutIntoCashInventoryPacket extends BaseCashOperationPacket {
   private final Integer cashId;

   private final Byte invType;

   public PutIntoCashInventoryPacket(Integer action, Integer cashId, Byte invType) {
      super(action);
      this.cashId = cashId;
      this.invType = invType;
   }

   public Integer cashId() {
      return cashId;
   }

   public Byte invType() {
      return invType;
   }
}
